.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,26 @@
+Make sure you have checked _all_ steps below.
+
+### Jira
+
+- [ ] My PR addresses the following [Parquet Jira](https://issues.apache.org/jira/browse/PARQUET/) issues and references them in the PR title. For example, "PARQUET-1234 My Parquet PR"
+  - https://issues.apache.org/jira/browse/PARQUET-XXX
+  - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
+
+### Tests
+
+- [ ] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:
+
+### Commits
+
+- [ ] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
```

Comment:
I don't think this is necessary, we always squash whole PR-s into single commits when merging them. Recommending squashing can even be counterproductive as authors may think they must squash before pushing updates to the review, which makes reviewing harder.

Reply:
Good point. I think this was still from the pre-gitbox era, where we couldn't use Github to squash the commits. I've removed the line regarding the squashing.